hack/test-go.sh

```diff
@@ -131,7 +131,7 @@ if [[ -n "${junit_report}" ]]; then
     os::util::ensure::built_binary_exists 'gotest2junit'
     report_file="$( mktemp "${ARTIFACT_DIR}/unit_report_XXXXX" ).xml"
 
-    go test -json ${gotest_flags} ${test_packages} 2>"${test_error_file}" | tee "${JUNIT_REPORT_OUTPUT}" | gotest2junit > "${report_file}"
+    go test -p 1 -json ${gotest_flags} ${test_packages} 2>"${test_error_file}" | tee "${JUNIT_REPORT_OUTPUT}" | gotest2junit > "${report_file}"
```

Comment:
presumably -p is already being set in gotest_flags so you can't set it again.

Reply:
@bparees Thanks!